Composer/packages/ui-plugins/composer/src/index.ts

```diff
@@ -142,7 +142,7 @@ const DefaultFormSchema: FormUISchema = {
     helpLink: 'https://aka.ms/bfc-using-http',
   },
   [SDKKinds.IfCondition]: {
-    label: () => formatMessage('Branch: If/Else'),
+    label: () => formatMessage('Branch: If/else'),
     subtitle: () => formatMessage('If Condition'),
```

Comment:
should this be `If condition` ?

Reply:
Fixed.

Reply:
I also made the rest of the subtitles consistent with the casing style.